monero-outreach-docs/translations/zh-hans/website-sections/aboutus_zh-hans.md

```diff
@@ -0,0 +1 @@

Our Roadmap  我们的路线图

FOUNDATION   根基
Create communication systems.  创建沟通系统。
Establish workflow.  建立工作流程。
Plan short-term and long-term goals.  设立短期及长期目标。
Develop website.   建设网站。
Survey current and potential users to gain insight that will inform future strategy. 
对现有及潜在客户进行调查，为未来战略洞察信息。

Attract talented volunteers and contributors.  吸引有才华的志愿者及贡献者。

Become familiar with the processes and people of the Monero project.   
熟悉门罗币项目的流程和人员

STRUCTURE  结构

Develop toolkits to aid in outreach; considering new and current users, merchants, investors, and journalists. 开发工具箱援助推广组; 支持新老用户、商家、投资者和记者。

Provide examples of how our efforts add value to the project and greater vision.
举例说明我们的努力如何为项目和更大的愿景增加价值。

Fundraise, secure sponsors, and create revenue internally to support additional professionals and long-term stability. 筹集资金，保护赞助商，并在内部创造收益，以支持更多的专业人士和长期稳定。

Research the individuals, groups, and publications that we want to reach.  
研究我们想接触的个人、团体和出版物。

Strengthen relationships between individuals and groups within the Monero project, finding ways to build team identity despite it’s decentralized nature.  
加强门罗币项目中个人和团队之间的关系，找到建立除去中心化以外的团队特性的方法。

OPERATIONAL 运作

Develop stories that will resonate with a wide audience and spur adoption. 
编写能够引起广大观众共鸣的故事，并鼓励人们接受。

Disperse original content and press releases to mainstream publications.   
将原创内容和新闻稿分发到主流出版物。

Attract professional writers, editors, policy experts, and Public Relations (PR) representatives who can maintain the ethos and values of Monero while bringing it to the public.  
吸引专业作家、编辑、政策专家和公共关系(PR)代表，他们能够在向公众传达时保持门罗币的精神和价值观。

Continue researching, surveying, and developing or expanding on toolkits for various audiences.   继续为广大用户研究、调查、开发或扩展工具包。

Explore creative PR stunts to help drive greater awareness and adoption of Monero.   
探索创造性的公关技巧，以帮助增加门罗币的认知度和接受度。
  
SOCIAL CHANGE 社会联结
Identify ways Monero can resonate well with policymakers and engage with them.
找出门罗币能与政策制定者产生共鸣之处并参与其中。

Identify ways Monero can resonate well with common people throughout the globe, particularly in areas with depressed economies or corrupt governments.
找出门罗币能够与全球普通民众产生共鸣的方式，尤其是在经济萧条或政府腐败的地区。

Identify ways Monero can resonate well with business and industrialists and engage with them.找出门罗币能与商界和实业家产生共鸣的方式，并与他们互动。

Continue researching, surveying, and developing or expanding on toolkits for various audiences.
继续为不同的用户研究、调查、开发或扩展工具包。

Explore new ways to safeguard and expand the use of Monero for achieving fundamental human rights.
 探索保护及拓展门罗币用途的新途径，以实现基本人权。

How to Contribute :  如何做贡献：

If you are interested and enthusiastic about supporting our efforts, here are the best ways to do so:
如果你有兴趣和热情支持我们的工作，以下是最好的方法:

1.
Join the conversation on Telegram:   加入我们Telegram讨论：
@monerooutreach

2.
Choose a task on Taiga and begin helping us out: 在Taiga上选择一个任务并开始帮助我们:
taiga.getmonero.org/project/xmrhaelan-monero-public-relations/


3.
Donate XMR:  捐赠门罗币

Copy to Clipboard    复制到键盘

```

Comment:
- Change the files name to "_**about_zh-hans.md**_".

Reply:
Done.